src/Test.FunctionalTests.Sarif/SarifValidatorTests.cs

```diff
@@ -18,7 +18,7 @@ namespace Microsoft.CodeAnalysis.Sarif
 {
     public class SarifValidatorTests
     {
-        public const string JsonSchemaFile = "sarif-schema.json";
+        public const string JsonSchemaFile = "sarif-schema-2.1.0-rtm.0.json";
```

Comment:
>2.1.0-rtm.0 [](start = 59, length = 11)

should we use VersionConstants.SchemaVersionAsPublishedToSchemaStoreOrg here? (and other files which use this literal) this way updating to newer release will be a few steps less.

Reply:
Good idea!

---
In reply to: [285310264](https://github.com/microsoft/sarif-sdk/pull/1493#discussion_r285310264) [](ancestors = 285310264)